defaults/test_default.yaml

```diff
@@ -160,6 +160,7 @@ mgmt_docker_image: ''
 k8s_log_api_calls: false
 k8s_deploy_monitoring: false
 k8s_minio_storage_size: '10Gi'
+k8s_n_scylla_pods_per_cluster: 3
```

Comment:
Fall-back for case later configurations get updated and if new K8S backend gets added. 
We have lots of options here which work as a fall-back. 
If you think it makes any harm then I can remove it.
Personally, I don't see any harm having it here.

Reply:
`n_db_nodes` by design doesn't have defaults, since we want each test case to be clear on the setup. 
since `k8s_n_scylla_pods_per_cluster` is it's equivalent for k8s, (and depends on  `n_db_nodes` in some why)

I think they should be treated the same.

also having multiple places for the default of the same value, could cause confusion on where to change for  changing this default. Right now it's or in the backend defaults or in the test_default

Reply:
So, it is not clear to me, do you insist on removing it or not?

Reply:
Leave it